apps/storybook-react/package.json

```diff
@@ -28,7 +28,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "storybook": "^8.3.5",
-    "tailwindcss": "^3.4.13",
+    "tailwindcss": "^3.0.0",
```

Comment:
Aligning tailwindcss version across the monorepo 

<img width="298" alt="Screenshot 2024-10-30 at 2 30 14 PM" src="https://github.com/user-attachments/assets/9f0e46bd-a605-4368-893d-f104b4dc3b13">